test/unit/serverStart.test.js

```diff
@@ -46,6 +46,7 @@ describe('Server Start', () => {
   })
 
   it('runs the server', async () => {
+    jest.setTimeout(10000)
```

Comment:
Can you clarify the need for these being added?

Reply:
Hi 

So jest has a default waiting time of 5seconds before a function is deemed to time out, on the pre-commit, the unit tests take too long and this fails the unit tests, preventing the commit.

After the above change, the server has enough time to startup and thus allowing the tests to pass.

I felt like this was wrong to commit, but I did it so that I could get feedback on the issue and then we can resolve accordingly.